project/src/main/ui/menu/PracticeMenu.tscn

```diff
@@ -73,9 +73,9 @@ __meta__ = {
 }
 
 [node name="Buttons" type="HBoxContainer" parent="VBoxContainer/Mode"]
-margin_left = 92.0
+margin_left = 88.0
```

Comment:
Why did these values change?

Reply:
These values are dictated by a LayoutManager, and it's possible the LayoutManager's behavior changed in a recent version of Godot.